src/js/components/DataFilter/__tests__/DataFilter-test.tsx

```diff
@@ -68,6 +68,48 @@ describe('DataFilter', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('select multiple options', () => {
+    const { container } = render(
+      <Grommet>
+        <Data
+          data={data}
+          properties={{
+            enabled: {
+              label: 'Enabled',
+              options: [
+                { label: 'Enabled', value: true },
+                { label: 'Disabled', value: false },
+              ],
+            },
+            'type.name': {
+              label: 'Type',
+            },
+          }}
+        >
+          <DataFilters>
+            <DataFilter property="name" />
+            <DataFilter property="enabled" />
+            <DataFilter
+              property="type.name"
+              options={[
+                { label: 'ZZ', value: 1 },
+                { label: 'YY', value: 2 },
+                { label: 'aa', value: 3 },
+                { label: 'bb', value: 4 },
+                { label: 'cc', value: 5 },
+                { label: 'dd', value: 6 },
+                { label: 'ee', value: 7 },
+                { label: 'ff', value: 8 },
+              ]}
+            />
+          </DataFilters>
+        </Data>
+      </Grommet>,
+    );
+
```

Comment:
Right now the snapshot isn't capturing information relevant to the fix made in this PR. I think before making the snapshot assertion we should fire an event to open the filters layer and then select one of the options for 'type.name' to filter by. Then close the drop and take a snapshot of the table.

Reply:
Thanks for clarifying! Will connect with you on Slack if I run into any questions.